client/my-sites/plugins/access-control.js

```diff
@@ -90,18 +102,24 @@ const hasRestrictedAccess = ( site ) => {
 		);
 	}
 
-	if ( abtest( 'businessPluginsNudge' ) === 'drake' && hasErrorCondition( site, 'noBusinessPlan' ) ) {
-		pluginPageError = {
-			title: i18n.translate( 'Want to add a store to your site?' ),
-			line: i18n.translate( 'Support for Shopify, Ecwid, and Gumroad is now available for WordPress.com Business.' ),
-			action: i18n.translate( 'Upgrade Now' ),
-			actionURL: '/plans/' + site.slug,
-			illustration: '/calypso/images/drake/drake-whoops.svg',
-			actionCallback: () => {
-				analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', { cta_name: 'business_plugins' } );
-			},
-			featureExample: getMockBusinessPluginItems()
-		};
+	if ( hasErrorCondition( site, 'noBusinessPlan' ) ) {
+		switch ( 'nudge' ) {
+			case 'nudge':
+				pluginPageError = {
+					abtest: 'nudge'
+				};
+				break;
+
+			case 'drake':
+			default:
+				pluginPageError = wpcomPluginPageError;
+				pluginPageError.actionUrl = '/plans/' + site.slug;
+				break;
+		}
+	}
+
+	if ( ! sites.hasSiteWithPlugins() && ! site ) {
```

Comment:
I think the order of the conditions should be reversed here - otherwise, if `site` is undefined, an exception will be thrown.